stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -308,6 +314,13 @@ public class MessageListViewModel(
                         } else {
                             newState
                         }
+
+                        messagesState.messageItems.firstOrNull {
+                            it is MessageItemState && it.message.id == scrollToMessage?.id
```

Comment:
What if we have thousands of messages in? For every emission, we'll iterate until we find the item we have (or not).

Maybe we can approach this in a different way -> 
- we can probably call focusMessage without it being in the list (we check if it exists in the list there)
- we can probably optimize the focusMessage function with a sorting/lookup algorithm to make it more performant